src/app/modules/gb-export-module/gb-export.component.ts

```diff
@@ -66,15 +66,19 @@ export class GbExportComponent implements OnInit, OnDestroy {
     this.exportService.exportJobName = value;
   }
 
-  get isTransmartEnv(): boolean {
-    let env = this.appConfig.getEnv();
-    return (env === 'default') || (env === 'transmart');
-  }
-
   get isExternalExportAvailable(): boolean {
     return this.appConfig.getConfig('export-mode')['name'] !== 'transmart';
   }
 
+  get isTransmartSurveyTableDataView(): boolean {
+    let exportMode =  this.appConfig.getConfig('export-mode');
+    return exportMode['name'] === 'transmart' && exportMode['data-view'] === 'surveyTable';
+  }
+
+  get isTransmartDateColumnIncluded(): boolean {
```

Comment:
perhaps whether to include dateColumn should also be assessed via config, just like isTransmartSurveyTableDataView

Reply:
hmm, but this is now defined via checkbox, which allows users to decide whether to include it or not. I guess what is missing here is a setter.

Reply:
it's added now